### PR TITLE
Bump v0.2.0+v0.25.0

### PR DESCRIPTION
**Pull request by arg0d**
Create `docs/RELEASE.md` containing the release procedure.

Refactor configuration options from `README.md` into `docs/CONFIGURATION.md`.

### CODE REVIEW

File: docs/CODE_OF_CONDUCT.md
Comment (Savolro) [inline docs/CODE_OF_CONDUCT.md:1]: Why was this moved?

Comment (arg0d) [reply docs/CODE_OF_CONDUCT.md:1]: I think the number of .md files in root is becoming too large with 5 files right now (6 including README.md which I left in root), and `docs` is the standard directory in Github https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes

File: docs/RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+upstream `uniffi-rs` version in the build metadata component (denoted as `+`). The upstream explicit
+upstream `uniffi-rs` aids consumer to target the same upstream version when mixing multiple
+generators, e.g. `uniffi-bindgen-cs` and `uniffi-bindgen-go`.
+```
+v0.2.0+v0.25.0
+```
+
+## Update version in [bindgen/Cargo.toml](../bindgen/Cargo.toml)
+
+Note that the version in [bindgen/Cargo.toml](../bindgen/Cargo.toml) does is not prefixed with `v`,
```
Comment (Lipt0nas) [inline docs/RELEASE.md:17]:
```suggestion
Note that the version in [bindgen/Cargo.toml](../bindgen/Cargo.toml) is not prefixed with `v`,
```

**Review by Lipt0nas — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -92,3 +55,18 @@ each generator targets the same `uniffi-rs` version.
 To simplify this choice `uniffi-bindgen-cs` and `uniffi-bindgen-go` use tag naming convention
 as follows: `vX.Y.Z+vA.B.C`, where `X.Y.Z` is the version of the generator itself, and `A.B.C` is
 the version of uniffi-rs it is based on.
+
+The table shows `uniffi-rs` version history for tags that were published before tag naming convention described above was introduced.
+
+| uniffi-bindgen-cs version                | uniffi-rs version                                |
```
Comment (Lipt0nas) [inline README.md:61]:
```suggestion
| uniffi-bindgen-go version                | uniffi-rs version                                |
```

**Review by Lipt0nas — APPROVED** [review]

**Review by Savolro — APPROVED** [review]
